### PR TITLE
DM-15208 Fix disabled skipTEx option

### PR DESCRIPTION
**Pull request by parejkoj**

### CODE REVIEW

File: python/lsst/validate/drp/matchreduce.py [hunk 1]
```diff
@@ -46,7 +44,7 @@
 
 
 def build_matched_dataset(repo, dataIds, matchRadius=None, safeSnr=50.,
-             useJointCal=False):
+                          useJointCal=False, skipTEx=False):
```
Comment (r-owen) [inline python/lsst/validate/drp/matchreduce.py:48]: Do you really want to force skipTEx to False here?

File: python/lsst/validate/drp/validate.py [hunk 2]
```diff
@@ -319,12 +321,11 @@ def add_measurement(measurement):
...
         adxName = 'AD{0:d}'.format(x)
 
-
         amx = measureAMx(metrics['validate_drp.'+amxName], matchedDataset, D*u.arcmin)
         add_measurement(amx)
 
-        afx_spec_set = specs.subset(required_meta={'instrument':'HSC'}, spec_tags=[afxName,])
-        adx_spec_set = specs.subset(required_meta={'instrument':'HSC'}, spec_tags=[adxName,])
+        afx_spec_set = specs.subset(required_meta={'instrument': 'HSC'}, spec_tags=[afxName, ])
+        adx_spec_set = specs.subset(required_meta={'instrument': 'HSC'}, spec_tags=[adxName, ])
```
Comment (r-owen) [inline python/lsst/validate/drp/validate.py:328]: Do you really want HSC hard-coded here? The value is `instrument` below. I realize this is pre-existing code.

File: python/lsst/validate/drp/matchreduce.py [hunk 3]
```diff
@@ -154,6 +148,11 @@ def _loadAndMatchCatalogs(repo, dataIds, matchRadius,
         calibration.
     matchRadius :  afwGeom.Angle(), optional
         Radius for matching. Default is 1 arcsecond.
+    useJointCal : bool, optional
+        Use jointcal/meas_mosaic outputs to calibrate positions and fluxes.
+    skipTEx : bool, optional
+        Skip TEx calculations (useful for older catalogs that don't have
+        PsfShape measurements).
```
Comment (r-owen) [inline python/lsst/validate/drp/matchreduce.py]: Please put classes in back tickets at least for new docs, e.g. `bool` -> `` `bool` ``

**Review by r-owen — APPROVED** [review]
In some cases I confess I couldn't really see what was going on, but overall it looked fine. I had a few minor questions and suggestions.

File: python/lsst/validate/drp/matchreduce.py (hunk 1 above)
Comment (parejkoj) [reply python/lsst/validate/drp/matchreduce.py:48]: Default for it is False everywhere, which is changed by the config option.

File: python/lsst/validate/drp/validate.py (hunk 2 above)
Comment (parejkoj) [reply python/lsst/validate/drp/validate.py:328]: I'm not going to modify the existing code other than to flake8 clean it: @SimonKrughoff or @wmwv would have to decide what appropriate defaults are here.

**Pull request merged**